### DISCUSSION TITLE
Installer for Windows?

### DISCUSSION THREAD

**Discussion by survivor303** [KDE/ghostwriter#829] [Ideas]
What Ghostwriter really need for now is installer for Windows, binaries are there yes, but when there are lot of files under it, you really cannot call it "portable".

When it gets installer, then it it is easier to add windows based package managers (winget, chocolatey), so there are benefits to use proper installer.

**Comment by wereturtle** [comment]
It's coming Soon (TM).  Now that the app is included under the KDE umbrella, we'll get the benefit of signed installers, both for individual download and for the Windows store for easy updates.

**Comment by HomerusJa** [comment]
Just asking, what's the progress on this? There seems still to be no installer...

I think this would be a fun little thing to work on. I would do it if there is any interest.

**Reply by survivor303** [reply]
If you have needed skills, please and contribute 👍

**Reply by HomerusJa** [reply]
I don't have them now, but I'll get on it soon!

**Reply by HomerusJa** [reply]
I've got working on it, but I have to admit that I don't really understand the release process on invent. I've got it kinda working on Github Actions either way using [this](https://github.com/AliceOh/CreateWindowsInstaller/).
